docs/user/aws/limits.md

```diff
@@ -53,8 +53,13 @@ the new VPC. If you intend to create more than 20 clusters, you will need to req
 
 ## Security Groups
 
-Each cluster creates 10 distinct security groups. The default limit of 2,500 for new accounts allows for many clusters
-to be created.
+Each cluster creates 4 distinct security groups. The default limit of 2,500 for new accounts allows for many clusters
+to be created. The three security groups created by the default install are:
+
+  1. VPC default
+  1. Master
+  1. Worker
+  1. Router/Ingress
```

Comment:
```
$ git grep '^resource "aws_security_group"' origin/pr/1306 | cat
origin/pr/1306:data/data/aws/bootstrap/main.tf:resource "aws_security_group" "bootstrap" {
origin/pr/1306:data/data/aws/vpc/sg-master.tf:resource "aws_security_group" "master" {
origin/pr/1306:data/data/aws/vpc/sg-worker.tf:resource "aws_security_group" "worker" {
```

So I'm not sure whether the VPC default counts or not (I'd guess it does), but *this* entry should probably be "Bootstrap", and we might want to list it before the master entry.